components/profile_popover/profile_popover.tsx

```diff
@@ -151,19 +158,19 @@ interface ProfilePopoverProps extends Omit<React.ComponentProps<typeof Popover>,
     enableLastActiveTime: boolean;
 
     timestampUnits: string[];
+
+    isCallsEnabled: boolean;
 }
 type ProfilePopoverState = {
     loadingDMChannel?: string;
+    show: boolean;
```

Comment:
The visibility of the profile popover isn't really handled by the popover itself, it's handled by whatever opened it. Using this `show` variable to hide the popover will lead to conflicts with the state inside and outside the popover. Try closing the popover with the X button and then opening it again; it results in a JavaScript error. 

Instead, to close the popover, you can simply call `this.props.hide()`. So I would do that in `handleClose` and remove the `show` variable.

Reply:
Good point @cyrilzhang-mm I update accordingly